libs/templates/404/404.js

```diff
@@ -1,13 +1,28 @@
-import { getConfig, createTag } from '../../utils/utils.js';
+import { getConfig, createTag, loadArea, getMetadata } from '../../utils/utils.js';
 import { sampleRUM } from '../../utils/samplerum.js';
 
-async function load404() {
-  const config = getConfig();
+const { base, contentRoot, locale } = getConfig();
+
+async function get404(path) {
+  const { prefix } = locale;
+  const href = path || `${base}${prefix}/fragments/404#_dnt`;
```

Comment:
To verify, `#_dnt` used because it's already pointing to the localized path?

Reply:
Yes. And the localized path is a bit non-standard... being after libs.